ui/js/index.js

```diff
@@ -15,120 +15,299 @@
+                ]
+            },
+            {
+                'id': 'discussions',
+                'title': oae.api.i18n.translate('__MSG__MY_DISCUSSIONS__'),
+                'icon': 'fa-comments',
+                'closeNav': true,
+                'layout': [
+                    {
+                        'width': 'col-md-12',
+                        'widgets': [
+                            {
+                                'name': 'discussionslibrary',
+                                'settings': {
+                                    'context': oae.data.me,
+                                    'canManage': true
+                                }
+                            }
+                        ]
+                    }
+                ]
+            },
+            {
+                'id': 'groups',
+                'title': oae.api.i18n.translate('__MSG__MY_GROUPS__'),
+                'icon': 'fa-group',
+                'closeNav': true,
+                'layout': [
+                    {
+                        'width': 'col-md-12',
+                        'widgets': [
+                            {
+                                'name': 'memberships',
+                                'settings': {
+                                    'context': oae.data.me,
+                                    'canManage': true
+                                }
+                            }
+                        ]
+                    }
+                ]
+            },
+            {
+                'id': 'network',
+                'title': oae.api.i18n.translate('__MSG__MY_NETWORK__'),
+                'icon': 'fa-random',
+                'closeNav': true,
+                'layout': [
+                    {
+                        'width': 'col-md-12',
+                        'widgets': [
+                            {
+                                'name': 'network',
+                                'settings': {
+                                    'context': oae.data.me,
+                                    'canManage': true
+                                }
+                            }
+                        ]
+                    }
+                ]
             }
+        ];
+
+        $(window).trigger('oae.trigger.lhnavigation', [lhNavPages, lhNavActions, '/']);
+        $(window).on('oae.ready.lhnavigation', function() {
+            $(window).trigger('oae.trigger.lhnavigation', [lhNavPages, lhNavActions, '/']);
+        });
+    };
+
+    /**
+     * Show the `preferences` widget if appropriate
+     */
+    var showPreferences = function() {
+        if ($.url().attr('query') === 'emailpreferences') {
+            $(document).trigger('oae.trigger.preferences');
         }
     };
 
-    renderLandingPage();
-    setUpSearch();
+
+    ////////////////////////
+    // GENERAL PAGE SETUP //
+    ////////////////////////
+    
+    // The basic index page varies based on whether the user
```

Comment:
`When the user is authenticated, their personal space will be loaded. When the user is anonymous, the tenant landing page will be shown.`